Pdf4QtLib/sources/pdfutils.cpp

```diff
@@ -564,4 +568,16 @@ QColor PDFColorScale::map(PDFReal value) const
     return QColor::fromRgbF(r, g, b);
 }
 
+QDataStream& operator<<(QDataStream& stream, long unsigned int i)
+{
+    stream << quint64(i);
+    return stream;
+}
+
+QDataStream& operator>>(QDataStream& stream, long unsigned int &i)
+{
+    stream >> reinterpret_cast<quint64&>(i);
```

Comment:
This code is potentially dangerous. I will prefer something like this:

quint64 value = 0;
stream >> value;
i = static_cast<long unsigned int>(value);

Reply:
ok